docbrown/core/src/tprop.rs

```diff
@@ -17,6 +19,13 @@ pub(crate) enum TProp {
     Bool(TCell<bool>)
 }
 
+impl Default for &TProp {
+    fn default() -> Self {
+        static DEFAULT: TProp = TProp::Empty;
```

Comment:
whaaaaat?

Reply:
In props.rs, line 108, among other places, we need to return the default value for &TProp. Tried returning &Default::default, but doesn't compile because the function owns the value out of Default::default(). Is there another way to fix this?